src/server/server.js

```diff
@@ -1,22 +1,28 @@
 import {SocketCluster} from 'socketcluster';
-import {auth0Secret} from './secrets';
 import path from 'path';
-// import os from 'os';
+import { getDotenv } from '../universal/utils/dotenv';
+
+// Import .env and expand variables:
+getDotenv();
+
+const clientSecret = process.env.AUTH0_CLIENT_SECRET ||
```

Comment:
I left the name of the environment variable the same to match the stupid field names from auth0 but changed the variable name in the source. Minor change, but optics matter.

I also grepped for the key and the varname in the build, it isn't exposed. If AUTH0_CLIENT_SECRET were declared in the EnvironmentPlugin, and if that environment variable were used in some universal code it would be.